src/screens/CardGameScreen/index.js

```diff
@@ -144,8 +149,14 @@ class CardGameScreen extends React.Component {
       };
     if (!this.state.recentProposals) {
       navigation.navigate('BatchResults', { screenProps });
-    } else {
-      this.props.setRecentProposalData(this.state.votes);
+      alert('batch results');
+    } else if (this.state.recentProposals) {
```

Comment:
If there's a need for two conditions I would prefer to have 2 if statements for ease of read

Reply:
Modified condition to more readable and comprehensive.